_posts/en/newsletters/2020-11-25-newsletter.md

```diff
@@ -0,0 +1,120 @@
+---
+title: 'Bitcoin Optech Newsletter #125'
+permalink: /en/newsletters/2020/11/25/
+name: 2020-11-25-newsletter
+slug: 2020-11-25-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces a website that tracks miner support for
+taproot activation and includes our regular sections with popular
+questions and answers from the Bitcoin StackExchange, announcements of
+releases and release candidates, and notable changes to popular Bitcoin
+infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Website listing miner support for taproot activation:** Bitcoin
+  mining pool [Poolin][] has created [a website][taprootactivation.com]
+  to help track miner support for activating
+  [taproot][topic taproot] (including [schnorr signatures][topic schnorr
+  signatures] and [tapscript][topic tapscript]).  As of this writing,
+  all pools listed on the site---who represent more than half of the
+  current network hash rate---have indicated that they will support
+  activation.  If this is reflective of the opinions of the remaining
+  pools, then it should be easy to accomplish the activation of taproot
+  after its implementation has been released and adopted by a moderate
+  number of users.  For additional information about the creation of the
+  website, see Bitcoin Magazine's [article][btcmag ta.com].
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update on Tuesday
+
+- [C-Lightning 0.9.2rc1][C-Lightning 0.9.2] is a release candidate for
```

Comment:
Can update this to https://github.com/ElementsProject/lightning/releases/tag/v0.9.2